### ISSUE TITLE
Data Quality Dashboard for DBpedia

### ISSUE THREAD

**Issue by mgns** [dbpedia/GSoC#2]
# Description
DBpedia offers large quantities of structured data. Though, DBpedia has partly insufficient data quality which originate from different sources, e.g. incorrect extractions and value transformations in the extraction framework, inconsistent mappings, incorrect data in Wikipedia articles, and generally incompleteness.
# Goals
Visualize a set of metrics in an easy to read interactive UI that facilitates the decision on what should be fixed next in DBpedia.
# Impact
The interface will help DBpedia contributors to adopt a “data quality first” attitude, enable data-driven prioritization of development tasks.
# Warm up tasks
 - Find 10 or 100 errors in DBpedia #11
 - Read about [Quality Assessment Methodologies for Linked Open Data](http://www.semantic-web-journal.net/system/files/swj414.pdf)
# Mentors
Possible names: _Bharat Suri_

# Keywords
data quality, front-end, full-stack, javascript, react js, meteor js, user interface, UI

**Comment by vedularaghu** [comment]
Hi, I'm an undergraduate student of CET, Bhubaneswar. I have prior experience in full stack web development (Node js , ReactJs, Javascript). I would like to work on this project. Please help in picking up issues to start with this project.
Thank you.

**Comment by vedularaghu** [comment]
Hi, I'm an undergraduate student of CET, Bhubaneswar. I have prior experience in full stack web development (Node js , ReactJs, Javascript). I would like to work on this project. Please help in picking up issues to start with this project.
Thank you.

**Comment by mgns** [comment]
Hi, there is a warm-up task (#11) referenced in the description. You could start with that.

**Comment by PseudoNerd** [comment]
I am Tanmay, I'm an undergraduate from India.

I'd like to work on this project for the coming summer of code.
 I had mentioned a query about the warmup-task associated with it and would like to take it up.

**Comment by LakshanSS** [comment]
Hi! I am Lakshan. I am interested in doing this project. I am familiar with ReactJs and JavaScript. I would like to work on this in GSOC 2019.

**Comment by beyzayaman** [comment]
Hi @PseudoNerd and @LakshanSS. If you try some of the warm up tasks and share with us (like found errors) it would be good to see it.

**Comment by akata01** [comment]
Hi, i am akata. I would like to work on this project for gsoc 2019. I will start with the error finding warmup task as mentioned by @beyzayaman .

**Comment by PseudoNerd** [comment]
I am currently pursuing a warm-up task and also taking a little more time with DBpedia. 
I'll be making a spreadsheet soon. 
Sorry for the delay.

**Comment by tramplingWillow** [comment]
Hello everyone, it is good that you have started working on the warm-up tasks. However, I would like to encourage you all to write your proposal for your chosen projects. This will allow us to review and suggest changes well before the submission deadline. Thanks!

**Comment by PseudoNerd** [comment]
I've already made a list of certain bugs in DBpedia but feel like the list doesn't fully cover DBpedia fully. Also, I've been working on the proposal for this project. 
Will it be okay to submit the mock proposal and the spreadsheet on 22nd of this month?
I have my exams going on and hence the delay.

**Comment by tramplingWillow** [comment]
That's okay, we look forward to going over the list you curate. You can share the spreadsheet with the appropriate links through mail, or you can simply share the document and we will leave our comments on it. I myself created a GitHub repository and shared the link with my mentors last year.

Yes, as the submission period begins from March 25th, it will be good to review your proposal before that so you can make a successful submission as soon as possible, hopefully well before the deadline i.e. April 9th.

**Comment by akata01** [comment]
Hey, @tramplingWillow , @beyzayaman i am working on the proposal for this project. I want to clear something about the tool for data quality assessment for this dashboard.
There are three data quality assessment tools mentioned in Quality assessment methodologies for Linked open data : 
1) Flemmings data quality assessment tool.
2) Sieve.
3) LODGRefine.

For this dashboard, are we supposed to use one or all of the above mentioned tools and visualize their outputs OR we will develop a new data quality assessment tool under this project?

If we are supposed to use one of the existing tools , I could not find flemmings data quality assessment tool by google search. It will be really helpful if you could share a link to the tool.

**Comment by beyzayaman** [comment]
Hi @akata01. You can just use one framework. The tools there are not so up-to-date. So I would suggest you look at [RDF Unit](https://github.com/AKSW/RDFUnit) which mainly used in DBpedia and forget about the others.

**Comment by tramplingWillow** [comment]
Hi everyone, please ensure that you submit a draft of your proposal for us to review and provide you with valuable feedback. Application period has already begun, so please plan accordingly.

**Comment by akata01** [comment]
Hi @mgns, @beyzayaman, @tramplingWillow! I have made a proposal for this project. I am sharing the pdf. Please if you could review it and provide me with valuable feedback.

[DBpedia GSoC proposal 2019(1).pdf](https://github.com/dbpedia/GSoC/files/3029171/DBpedia.GSoC.proposal.2019.1.pdf)

**Comment by akata01** [comment]
Hi @mgns, @tramplingWillow, @beyzayaman! I have made a proposal for this project. Should I submit it directly through the GSoC proposal submission page or do I need to send it to you through a mail or any other way you suggest so that you can review and it and provide me with valuable feedback?

**Comment by beyzayaman** [comment]
Hi @akata01. Maybe @tramplingWillow  and I can do it. Can you please first check if your proposal includes all of the sections from [this](http://tommaso-soru.it/files/misc/Akshay-DBpedia-GSoC-2017-proposal.pdf) successful proposal? Secondly, I will ask you to share with us your proposal in google docs or any editable format so that we can make some comments on it (it can be private mails as well)?

**Comment by akata01** [comment]
@beyzayaman I have followed the template given here -> https://wiki.dbpedia.org/gsoc-2019.Yes it includes all the sections from the mentioned proposal. I can add you to the proposal's google sheet. Please if you could share your email.

**Comment by beyzayaman** [comment]
yaman@infai.org

**Comment by akata01** [comment]
@beyzayaman here is the link to the google sheet -> https://docs.google.com/document/d/1ED1xtIsbUATLPn7M6PIuuxqcWQlHh4sv4hx4UwgecSU/edit?usp=sharing

**Comment by akata01** [comment]
@beyzayaman I have sent you an email with the GSoC proposal google sheet link. (I have deleted the previous comment with the link from here.)

**Comment by akata01** [comment]
@mgns @beyzayaman @tramplingWillow I saw this(@PseudoNerd ) malicious copying my GSoC proposal document. I am uploading the screenshots. I saw him editing and when I asked him if the comments chat, he quickly copied the entire. I have screenshots. I am uploading them here.

file:///home/akata/Pictures/gsoc)copy.png
![copying](https://user-images.githubusercontent.com/34716955/55511637-1490db00-562f-11e9-9e13-6aa7e4353067.png)

**Comment by PseudoNerd** [comment]
@akata01, First of all, I shouldn't have this link in the first place and it was public. 
Second, If you think your proposal was maliciously copied by me or in any way used, then the mentors can have a look at it themselves. 

Also, I've been working on this proposal for a very long time and this wouldn't help my cause. 

@mgns @tramplingWillow @beyzayaman , I'll get done with my proposal by tonight and you could check for anything malicious that I'm accused of.

**Comment by akata01** [comment]
@PseudoNerd @mgns @beyzayaman @tramplingWillow i can prove that @PseudoNerd  copied my idea and not just by screenshots but with facts. 

You just said that you are working on this proposal for a very long time but have you forked the RDFUnit tool repository 2 days ago only. Let me tell you why. Because you read my proposal (i will get to that how you have that link in the first place) and then only you knew to use that tool in the first place. If anyone wants to check can check that @PseudoNerd forked the RDFUnit tool repository on 1st April only which is the date on which i shared my proposal with the mentors with the shareable link.

Now he might go and delete that repository as soon as he reads this just link he deleted the comment from google docs chat box where after confronting him about his actions he first said that he didn't do anything wrong but then I told him that I have screenshots he confessed that he copied the entire proposal from top to bottom but soon deleted the comment. @PseudoNerd you confessed that you copied the proposal right ?

NOw how @PseudoNerd got the link in the first place. I pasted the link here in the issue comment section for mentors to review the proposal. As @PseudoNerd is subscribed to this issue he would have received the email with the link. But what he did wrong was he started using it and reading it to make his own proposal. And as I saw him making changes(inserting deleting things) to the proposal I confronted him and then he literally copied the entire thing in front of my eyes just after I sent him the message which I took a screenshot of posted above.
